jwt/src/main/java/me/uport/sdk/jwt/JWTTools.kt

```diff
@@ -32,12 +31,25 @@ import org.kethereum.extensions.toBigInteger
 import org.kethereum.hashes.sha256
 import org.kethereum.model.PUBLIC_KEY_SIZE
 import org.kethereum.model.PublicKey
-import org.kethereum.model.SignatureData
 import org.walleth.khex.clean0xPrefix
 import org.walleth.khex.hexToByteArray
 import java.math.BigInteger
 import java.security.SignatureException
 
+/**
+ * Method signature for the verification methods.
+ *
+ * @param publicKeys : List<PublicKeyEntry> list of public key entries to be verified against the signature
+ * @param signatureBytes : ByteArray the algorithm-specific encoded signature
+ * @param signedData : ByteArray the blob of data that was signed
+ *
+ */
+typealias VerificationMethod = (
```

Comment:
This defines the method signature for the supported verification methods.

Note that the signature is sent in raw form to allow the use of other signature schemes that may not conform to the currently used encodings.

Each verification method is now responsible to decode the signature according to its own algorithm